### ISSUE TITLE
[FEATURE] Tidy 5e Counters Integration

### ISSUE THREAD

**Issue by adriangaro** [Larkinabout/fvtt-custom-dnd5e#114]
**Describe your idea**
Atm cannot see the counters anywhere inside a Tidy 5e Sheet, any chance of getting integration for it?

**Versions**
- Module: 1.0.5
- System: dnd5e 3.3.1
- Foundry VTT: 12.327

**Additional context**
None

**Comment by Larkinabout** [comment]
Yes, there's some work to be done to make it possible, so it might not be a quick thing.

Blocked by #98.

**Comment by adriangaro** [comment]
Hey there, is there any update on this?

**Comment by kgar** [comment]
I'm really late to this, but I wanted to say thank you!
